src/Blockcore.Dns/appsettings.json

```diff
@@ -21,10 +21,12 @@
   "Dns": {
     "ListenPort": "53",
     "IntervalMin": "5",
-    "EndServerIp": "192.168.1.1"
+    "EndServerIp": "192.168.1.1",
+    "Identities": [
+      "did:is:03e9bcd2f568bddec7c2f36132ce4017e6a98077a9bc4ad7105c48b12f7b9ec7cb,",
```

Comment:
There is a bug here, an extra "," added at the end.

Reply:
ah ok well spotted, this is of course a fake did but its still a good example